auditbeat/module/auditd/audit_linux.go

```diff
@@ -164,14 +164,30 @@ func closeAuditClient(client *libaudit.AuditClient, log *logp.Logger) {
 	}
 }
 
+func getStatus() (*libaudit.AuditStatus, error) {
+	client, err := libaudit.NewAuditClient(nil)
+	defer func() {
+		if client != nil {
+			client.Close()
```

Comment:
I assume that if we move the `defer` statement to after the error check, we don't need the nil check, and we can just do `defer client.close()`

Reply:
Ah, makes sense. Thanks.

Reply:
a6b5df3